### ISSUE TITLE
transfer old issues to new repository

### ISSUE THREAD

**Issue by mfrasca** [WLOGSolutions/r-logging#10]
I've got a few issues in my repository, which I am not going to handle, and which have very possibly been handled already.  I wish to move them to the WLOGSolutions repository, and in my repository add a single issue like "please do not open issues here", with a link to here.
I tried to do that from the `transfer issue` functionality, but no, your repository does not show.
if you make me (temporarily) a member of your organization, I should be able to do that.
alternatively, review the issues in my repository, and advise me whether they can be closed.
I prefer the first option though.  more visible to all.
![image](https://user-images.githubusercontent.com/122869/62415471-b92c2500-b5ef-11e9-9b91-6953746f3533.png)

**Comment by ws171913** [comment]
Hello,

You should receive invitation to join WLOG team. Make me know, please, then
you will not need access any more.

Best regards,
Walerian Sokołowski

sob., 3 sie 2019 o 20:09 Mario Frasca <notifications@github.com> napisał(a):

**Comment by mfrasca** [comment]
thank you, I will look into it as soon as I have the chance.  I tried moving issues but seems that can only be across repositories with same owner.  Other option is to transfer the repository to your organization, but you already have it, with a clashing name.  or I'll look for an external tool for moving issues.  let me know if you have any preferences or suggestions.

**Comment by mfrasca** [comment]
I have the following two options, if you're interested in getting the issues on your fork:

- I rename my repository
- move to your organization
- migrate the issues

or

- we merge the your fork onto my repository
- we rename / delete your fork
- I move my repository to your organization
- I fork your repository and make clear that yours is the leading copy.

or

- I close all issues

I just did something similar as the second option, with mfrasca/ghini which became Ghini/server.
the second option has the advantage that you would get the 'reference' copy.

**Comment by ws171913** [comment]
Mario,

I also think the second option is optimal. Can you do it? Or you need help?

**Comment by mfrasca** [comment]
since you put me (temporarily) in your organization, I can do that.
it's possibly one hour work, maybe just a bit more.
I'll try to make the time today (being it a Sunday, I think it's the best option for you).

**Comment by mfrasca** [comment]
nope, the 'Settings' tab does not appear here, so I can't rename your fork.
I would not want to rename the original just to move it to your organization,
I prefer to do it as described.

**Comment by ws171913** [comment]
Mario,

I've renamed the fork to r-logging-wlog.

Best regards,
Walerian Sokołowski

niedz., 11 sie 2019 o 17:45 Mario Frasca <notifications@github.com>
napisał(a):

**Comment by mfrasca** [comment]
![image](https://user-images.githubusercontent.com/122869/62884764-eddd6200-bcfc-11e9-87b8-bf7a482a866b.png)
pfffffffffffff
more complicated than necessary, …, or maybe just as complicated as unavoidable.
let me know what you think of it.

**Comment by ws171913** [comment]
Mario,

I see that you have merged all the changes into base repository. Perhaps It
would be wiser to just remove r-logging from WLOGSolutions and transfer it
from your account?
That way issues on WLOG r-logging will be lost, but it is only one that
really matters. It can be just restored after transfer.

This issue consumes to much time already :(

pon., 12 sie 2019 o 19:31 Mario Frasca <notifications@github.com>
napisał(a):

**Comment by mfrasca** [comment]
> Perhaps It would be wiser to just remove r-logging from WLOGSolutions and transfer it from your account?

maybe yes.

it's not "consuming" much time, but yes it is taking time.

the fastest would quite obviously be that you make me "owner" of your organization for the time needed to perform this change.  but do you trust me that you have to decide.

**Comment by ws171913** [comment]
Here you are. Projects on github are public and opensource so I do not
think it is a big deal. Just make me know then you are finished I'll revoke
permissions.

śr., 14 sie 2019 o 15:57 Mario Frasca <notifications@github.com> napisał(a):
